hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/DBCheckpointServlet.java

```diff
@@ -195,7 +195,11 @@ private void generateSnapshotCheckpoint(HttpServletRequest request,
               .filter(s -> s.endsWith(ROCKSDB_SST_SUFFIX))
               .distinct()
               .collect(Collectors.toList()));
-      LOG.info("Received excluding SST {}", receivedSstList);
+      LOG.info("Received excluding SST {}. The total excluded sst files count is {}",
+          receivedSstList.subList(0, Math.min(5, receivedSstList.size())), receivedSstList.size());
+      if (LOG.isDebugEnabled()) {
+        LOG.debug("Received excluding SST {}", receivedSstList);
+      }
```

Comment:
nits:

1. Put quick info at the start, details later.  In this specific case: show count first, items afterwards.  Something like: `"Received list of {} SST files to be excluded, sample: {}", count, sublist)`
2. Avoid duplication.  Please extract and reuse logic for getting the sublist.
3. I think logging one message, debug _or_ info, is enough.  Message can be almost the same, just omit `, sample`.  Construct the message by extracting the common parts.